types/components/AccordionToggle.d.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import Button from './Button';
+import { BsPrefixComponent } from './helpers';
+
+export interface AccordionToggleProps {
+  eventKey: string;
+  onClick?: (arg0?: React.SyntheticEvent) => any;
+}
+
+declare class AccordionToggle<
+  As extends React.ReactType = typeof Button
```

Comment:
currently this is actually `'button'`, not our `Button`

Reply:
in the accordion example we're using `Button`
https://github.com/react-bootstrap/react-bootstrap/blob/8819727d423684bb69b0b45dd7b4a97a77d715f3/www/src/examples/Accordion/Basic.js#L4

Reply:
right, but the actual default is `button`: https://github.com/react-bootstrap/react-bootstrap/blob/8819727d423684bb69b0b45dd7b4a97a77d715f3/src/AccordionToggle.js#L23

though as noted in that PR, we should consider changing this to `<SafeAnchor>`